website/docs/r/guardduty_filter.html.markdown

```diff
@@ -0,0 +1,75 @@
+---
+layout: "aws"
+subcategory: "GuardDuty"
+page_title: "AWS: aws_guardduty_filter"
+description: |-
+  Provides a resource to manage a GuardDuty filter
+---
+
+# Resource: aws_guardduty_filter
+
+Provides a resource to manage a GuardDuty filter.
+
+## Example Usage
+
+```hcl
+resource "aws_guardduty_filter" "MyFilter" {
+  name        = "MyFilter"
+  action      = "ARCHIVE"
+  description = "Some description"
+  detector_id = "123456271278c0df5e089123480d8765"
```

Comment:
This could show a resource reference. Optionally, the `aws_guardduty_detector` could be included, but it's not necessary
```suggestion
  detector_id = aws_guardduty_detector.example.id
```